ampd/src/handlers/evm_verify_worker_set.rs

```diff
@@ -0,0 +1,124 @@
+use error_stack::ResultExt;
+use serde::Deserialize;
+
+use axelar_wasm_std::voting::PollID;
+use events_derive::try_from;
+
+use crate::types::{EVMAddress, Hash, TMAddress, U256};
+
+#[derive(Deserialize, Debug)]
+pub struct Operators {
```

Comment:
what's the difference between a worker and operator?

Reply:
My understanding is that workers are anyone running external services (voting, signing, maybe others as well) and operators are those signer addresses set in the EVM gateway contract. I don't like these names either, but I'm following the cosmowasm contracts' events for now.